4.7.2-windowsservercore-ltsc2016/sdk/Dockerfile

```diff
@@ -13,11 +13,11 @@ RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; `
 RUN Invoke-WebRequest -UseBasicParsing https://download.visualstudio.microsoft.com/download/pr/12210068/8a386d27295953ee79281fd1f1832e2d/vs_TestAgent.exe -OutFile vs_TestAgent.exe; `
     Start-Process vs_TestAgent.exe -ArgumentList '--quiet', '--norestart', '--nocache' -NoNewWindow -Wait; `
     Remove-Item -Force vs_TestAgent.exe; `
-# Install VS Build Tools
+    # Install VS Build Tools
     Invoke-WebRequest -UseBasicParsing https://download.visualstudio.microsoft.com/download/pr/12210059/e64d79b40219aea618ce2fe10ebd5f0d/vs_BuildTools.exe -OutFile vs_BuildTools.exe; `
     # Installer won't detect DOTNET_SKIP_FIRST_TIME_EXPERIENCE if ENV is used, must use setx /M
     setx /M DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1; `
-    Start-Process vs_BuildTools.exe -ArgumentList '--add', 'Microsoft.VisualStudio.Workload.MSBuildTools', '--add', 'Microsoft.VisualStudio.Workload.NetCoreBuildTools', '--quiet', '--norestart', '--nocache' -NoNewWindow -Wait; `
+    Start-Process vs_BuildTools.exe -ArgumentList '--add', 'Microsoft.VisualStudio.Workload.MSBuildTools', '--add', 'Microsoft.VisualStudio.Workload.NetCoreBuildTools', '--add', 'Microsoft.Net.Component.4.7.1.SDK', '--add', 'Microsoft.Component.ClickOnce.MSBuild', '--quiet', '--norestart', '--nocache' -NoNewWindow -Wait; `
```

Comment:
Seems like this should be installing `Microsoft.Net.Component.4.7.2.SDK` vs `Microsoft.Net.Component.4.7.1.SDK` since it is the 4.7.2 image.  This applies to the other 4.7.2 images as well.

Reply:
It's been a while, but I seem to recall that that doesn't actually work. The component doesn't exist in the 4.7.2 SDK. (Or didn't, anyway.)

Reply:
It is listed here - https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2017.  I verified it works.  If you have troubles let me know.

Reply:
Update: sure enough, it does exist now. I'll update all of these.